src/main/java/org/isf/hospital/model/Hospital.java

```diff
@@ -81,6 +86,14 @@ public class Hospital extends Auditable<String> {
 	@Column(name="HOS_CURR_COD")
     private String currencyCod;
 
+	@NotNull
+	@Column(name="HOS_VISITS_START")
+	private Time visitStartTime;
+
+	@NotNull
+	@Column(name="HOS_VISITS_END")
+	private Time visitEndTime;
+
 	@Version
 	@Column(name="HOS_LOCK")
     private Integer lock;
```

Comment:
I know that only some lines are just unchanged, but it seems we have a problem of indentation in some files.